core/src/main/java/org/apache/accumulo/core/iterators/IteratorUtil.java

```diff
@@ -235,18 +235,27 @@ private static void mergeOptions(Map<String,Map<String,String>> ssio, Map<String
   public static <K extends WritableComparable<?>,V extends Writable> SortedKeyValueIterator<K,V> loadIterators(IteratorScope scope,
       SortedKeyValueIterator<K,V> source, KeyExtent extent, AccumuloConfiguration conf, List<IterInfo> ssiList, Map<String,Map<String,String>> ssio,
       IteratorEnvironment env, boolean useAccumuloClassLoader) throws IOException {
-    List<IterInfo> iters = new ArrayList<>(ssiList);
-    Map<String,Map<String,String>> allOptions = new HashMap<>();
-    parseIteratorConfiguration(scope, iters, ssio, allOptions, conf);
-    return loadIterators(source, iters, allOptions, env, useAccumuloClassLoader, conf.get(Property.TABLE_CLASSPATH));
+
+    return loadIteratorsHelper(scope, source, extent, conf, ssiList, ssio, env, useAccumuloClassLoader, null);
```

Comment:
Related comment to below: would recommend switching the `null` back to `conf.get(Property.TABLE_CLASSPATH)`